src/prefect/flow_runs.py

```diff
@@ -114,17 +118,37 @@ async def main(num_runs: int):
 
             ```
     """
+    if poll_interval is not None:
+        get_logger().warning(
+            "The `poll_interval` argument is deprecated and will be removed in a future release. "
+        )
+
     assert client is not None, "Client injection failed"
     logger = get_logger()
+
+    filter = EventFilter(
```

Comment:
tiny nit to avoid overlap w soft keyword
```suggestion
    event_filter = EventFilter(
```